doc/sphinx/source/recipes/recipe_sea_surface_salinity.rst

```diff
@@ -0,0 +1,86 @@
+.. _recipes_sea_surface_salinity:
+
+Sea Surface Salinity Evaluation
```

Comment:
I think it would be good to have a section in the documentation on:

(1) how to obtain the shapefile needed
(2) in which directory to put the shapefile
(3) available regions (or at least examples) for analysis
(4) required / recommended preprocessor settings (extract_shape)